data/mods/No_NPC_Needs/effects_on_condition.json

```
@@ -0,0 +1,15 @@
+[
+  {
+    "type": "effect_on_condition",
+    "id": "reset_needs",
+    "recurrence": "4 hours",
+    "global": true,
+    "run_for_npcs": true,
+    "effect": [
+      { "arithmetic": [ { "u_val": "stored_kcal_percentage" }, "=", { "const": 25 } ] },
+      { "arithmetic": [ { "u_val": "thirst" }, "=", { "const": 0 } ] },
+      { "arithmetic": [ { "u_val": "fatigue" }, "=", { "const": 0 } ] },
+      { "arithmetic": [ { "u_val": "sleep_deprivation" }, "=", { "const": 0 } ] }
```

I'd remove this line because currently NPCs do a great job of finding a bed and sleeping when they need to. It's just eating and drinking that are problems.
```suggestion
      { "arithmetic": [ { "u_val": "sleep_deprivation" }, "=", { "const": 0 } ] }
```

---

Yeah, that was just me leaving it in there when I ~~copied~~ borrowed from the portal resurrection EOC for the purposes of testing because I was too lazy to use the debug mutation

---

No shame in being a cargo cultist.  I'm a priest of the cargo cult. :)

---

Praise the cargo cult